Rakefile

```
@@ -20,18 +20,20 @@
 require "date"
 require "tmpdir"
 
+VERSION_FULL = File.read("version_full")
+
 def env_var(name, default=nil)
   value = ENV[name] || default
   raise "${#{name}} is missing" if value.nil?
   value
 end
 
 def version
-  env_var("VERSION", File.read("version_full"))
+  env_var("VERSION", VERSION_FULL)
 end
 
 def new_version
-  env_var("NEW_VERSION")
+  env_var("NEW_VERSION", version.succ)
```

```suggestion
  @new_version ||= env_var("NEW_VERSION", version.succ)
```

`new_version` を何回も呼び出すので、↑みたいなことをしたいのですが、トップレベルでもやって良いもんですか？

---

動くんだけど行儀が悪いかな。

軽い処理だからキャッシュしなくてもいいよ。

マジメにやるなら https://github.com/groonga/groonga.org/blob/gh-pages/release_task.rb みたいにクラスを作るかな。

---

あざます。了解であります。